.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Publish
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  package:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        node-version: [20.11.0]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - name: Setup Chrome
+        uses: browser-actions/setup-chrome@v0.0.0
```

Comment:
Do we need to setup Chrome for this? I am not that familiar with the actions so just wandering.

Reply:
No, we don't, I copy-pasted too much. The checkout action does not need Chrome.